README.md

```diff
@@ -31,16 +31,19 @@ $ ./setup
 
 #### Running
 
+To run Rosie, you need to select a module to be called.
+For example, if you want to run `chamber_of_deputies` module, you should run this command:
```

Comment:
No, in a talk with @cuducos we decided that is better to run a specific module every time, because it will take to much time running more than one module.